pkg/repository/udmrepo/kopialib/backend/common.go

```diff
@@ -33,8 +33,7 @@ import (
 )
 
 const (
-	maxDataCacheMB         = 2000
-	maxMetadataCacheMB     = 2000
+	defaultCacheLimitMB    = 5000
```

Comment:
Curious about how we ended up with this default value ?

Reply:
No special reason, there is not a universal value which works for most of the cases.